tests/Elastic.CommonSchema.Tests/Repro/GithubIssue29.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Xunit;
+
+namespace Elastic.CommonSchema.Tests.Repro
+{
+	public class GithubIssue29
+	{
+		[Fact]
+		public void Reproduce()
+		{
+			// Metadata properties with null values should not be serialised
```

Comment:
I wonder if this is the proper fix for https://github.com/elastic/ecs-dotnet/issues/29, given that metadata is a dictionary I think we should emit `"prop" : null` instead.

Reply:
Good idea, fixed.